util_scripts/build-node.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pushd "`dirname $0`/.."
+    PATH_FULL=`pwd -P`
+popd
+LIBRARY_PATH="${PATH_FULL}/android/app/libs/Mysterium.aar"
+
+pushd $GOPATH/src/github.com/mysteriumnetwork/node
+    #bin/package_android amd64
+    cp -p build/package/Mysterium.aar $LIBRARY_PATH
+popd
```

Comment:
You can set-up WebStorm to always ensure newline at the end of file to avoid these red markers.